core/shared/src/main/scala/zio/Ref.scala

```diff
@@ -136,12 +226,13 @@ final class Ref[A] private (private val value: AtomicReference[A]) extends AnyVa
 
   /**
    * Atomically modifies the `Ref` with the specified partial function.
-   * if the function is undefined in the current value it returns the old value without changing it.
+   * If the function is undefined on the current value it returns the old value
+   * without changing it.
    *
    * @param pf partial function to atomically modify the `Ref`
    * @return `UIO[A]` modified value of the `Ref`
    */
-  def updateSome(pf: PartialFunction[A, A]): UIO[A] = IO.effectTotal {
+  def updateAndGetSome(pf: PartialFunction[A, A]): UIO[A] = IO.effectTotal {
```

Comment:
How about `updateSomeAndGet`? It feels more logical.

Reply:
Sure.